<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify</class>
 <widget class="QWidget" name="modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>701</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="submit">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_21">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="return_2">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>701</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <spacer name="horizontalSpacer_27">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_28">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>701</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLabel" name="label2">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性2：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label3">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性3：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="lineEdit4">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label6">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性3：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit1">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit2">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label1">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性1：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit3">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label5">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性2：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="lineEdit6">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label4">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性1：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit5">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
